mrobpy/SE3py.cpp

```diff
@@ -37,8 +37,9 @@ using namespace mrob;
 
 
 void init_geometry(py::module &m) {
-    py::class_<SE3>(m, "SE3")
-		.def(py::init<>(),
+    py::class_<SE3> se3(m, "SE3");
+
+    se3.def(py::init<>(),
```

Comment:
In lines 91-99, SE3() is used as the default argument. According to the [documentation](https://pybind11.readthedocs.io/en/stable/advanced/functions.html?highlight=py%3A%3Aarg#default-arguments-revisited):

> pybind11 must already be set up to deal with values of the type SomeType (via a prior instantiation of py::class_<SomeType>), or an exception will be thrown.

Therefore, you need to declare class SE3 separately from methods. In Windows without this change, errors occur during import of built library.